src/divider/Divider.tsx

```diff
@@ -3,8 +3,11 @@ import cc from 'classcat'
 
 export interface DividerProps {
   readonly className?: string
+  readonly inBorderedComponent?: boolean
 }
 
 export const Divider = ({ className }: DividerProps) => (
-  <div className={cc(className)} aria-hidden="true" />
+  <div className={cc(className)} aria-hidden="true">
+    <hr />
```

Comment:
can't you use _only_  `hr`?

Reply:
You answered your own question :D  https://github.com/blablacar/ui-library/pull/777#discussion_r448848822